### DISCUSSION TITLE
Hidden Channels messages cannot be loaded

### DISCUSSION THREAD

**Discussion by Blaze-arch-creator** [JustOptimize/return-ShowHiddenChannels#59] [Q&A] [answered]
when i opened the hidden channel, it looks like this
![Screenshot (224)](https://user-images.githubusercontent.com/114347591/198942327-d4ac7c30-307e-4189-bd23-c49a61f904d8.png)
someone can tell me why this happen? :0

**Comment by dimaguy** [comment, marked as answer]
You don't think that you can actually access/read hidden channels do you?  
Just select a channel that is not hidden, since this plugin does not yet have the safeguard set against people trying to get in the hidden ones
